VMEncryption/main/handle.py

```diff
@@ -180,7 +180,15 @@ def update_encryption_settings():
     logger.log('Updating encryption settings')
 
     # re-install extra packages like cryptsetup if no longer on system from earlier enable
-    DistroPatcher.install_extras()
+    try:
+        DistroPatcher.install_cryptsetup()
```

Comment:
Why are we not installing all extra packages here? (why just cryptsetup?)

Reply:
I am also curious why this changed from install_extras to install_cryptsetup?

Reply:
One idea:  For dual pass we may also need to install packages to support AAD auth.  On single pass this is not necessary and only needs crypt_setup.  So this line might have been inadvertently copied over from SP change?